lib/qtermwidget.cpp

```diff
@@ -749,3 +749,16 @@ int QTermWidget::getMargin() const
 {
     return m_impl->m_terminalDisplay->margin();
 }
+
+void QTermWidget::saveHistory(QIODevice *device)
+{
+    QTextStream stream(device);
+    PlainTextDecoder decoder;
+    decoder.begin(&stream);
+    m_impl->m_session->emulation()->writeToStream(&decoder, 0, m_impl->m_session->emulation()->lineCount());
+}
+
+void QTermWidget::sendCommand(const QString &command)
+{
+    m_impl->m_session->emulation()->sendText(command);
```

Comment:
Well, this is basically the same as `QTermWidget::sendText()`.

Reply:
It is indeed. I've just removed it now. Thanks